alwaysontop/render/index.js

```diff
@@ -293,9 +318,10 @@ class AlwaysOnTop extends EventEmitter {
 *
 * @param {JitsiIFrameApi} api - the Jitsi Meet iframe api object.
 * @param {Logger} loggerTransports - external loggers
+* @param {boolean} showOnPrejoin - whether AOT should show on prejoin screen
 */
-module.exports = (api, loggerTransports) => {
+module.exports = (api, loggerTransports, showOnPrejoin) => {
```

Comment:
let's do an options object before this gets out of hand...

Reply:
with keeping backwards compat? it would be a bit tricky, as I don t see an easy way to discern whether the caller used the new or old signature for it.

Reply:
something along the lines `module.exports = (options) { if (options.hasOwnProperty('api'))...` ? as api would be a mandatory prop.

Reply:
No I mean add the new option inside an options object...

Reply:
oh got it